### ISSUE TITLE
Continued use of http

### ISSUE THREAD

**Issue by cmungall** [identifiers-org/registry#6]
Many triplestores use identifiers.org http URIs. If identifiers.org were to stop supporting http and force a move to https, this would cause churn, since these triplestores would have to update. This would likely not be synced, meaning many federated queries would break.

It would be useful if id.org were to give a formal guarantee of continued support of the existing URLs at the string-match level, similar to what uniprot has done for purl.uniprot.org and what OBO does for obolibrary PURLs.

**Comment by sarala** [comment]
Hi Chris,
Yes, we are aware of this issue and we will continue to support both URLs. I will discuss with the wider team about releasing a formal statement in the coming weeks. Are there discussions around moving to 'https' at some point? When you introduce new datasets will you be using 'https'?

**Comment by cmungall** [comment]
No, we would like to continue to use http, and having a mix would be 
confusing.

I want to make sure that people using id.org URIs in their triplestores 
use the exact same string, otherwise joins fail

**Comment by zmughal** [comment]
Is there any update on this now?

@cmungall, I remember seeing your post on the semantic-web mailing list <https://lists.w3.org/Archives/Public/semantic-web/2023Jun/0019.html>, <https://github.com/OBOFoundry/purl.obolibrary.org/issues/705>.

Here I see both `http` and `https` being used for terms <https://docs.identifiers.org/pages/faq.html#url_structure>, <https://docs.identifiers.org/pages/identification_scheme.html>.
